tree/ntuple/v7/inc/ROOT/RDaos.hxx

```diff
@@ -40,6 +40,46 @@ namespace ROOT {
 namespace Experimental {
 namespace Detail {
 
+std::string inline GetDaosError(int err)
+{
+   return d_errstr(err);
+}
+
+struct RDaosObjectId {
+   using ObjectIndex_t = std::uint64_t;
+   daos_obj_id_t fData;
+   RDaosObjectId(std::uint64_t ntupleIndex, ObjectIndex_t objectIndex)
+      : fData{static_cast<decltype(daos_obj_id_t::lo)>(objectIndex),
+              static_cast<decltype(daos_obj_id_t::hi)>(ntupleIndex)}
+   {
+   }
+   [[nodiscard]] daos_obj_id_t Get() const { return fData; }
+};
+
+struct RDaosKey {
```

Comment:
Not directly related to this PR, as the type was already there: this data structure seems to be containing more than just the "key".  Perhaps a name like `RDaosBlobLocator` is more appropriate; what do you think?